uint8 ssp_index
uint8 siren_use
uint8 lights_use
uint8 multi
uint8 resposne_type
st_PRIV_EVENTS[] events
